data-access-services/transactions-service/src/main/java/at/deloittedigital/core_banking/transactions/controller/TransactionController.java

```diff
@@ -0,0 +1,26 @@
+package at.deloittedigital.core_banking.transactions.controller;
+
+import at.deloittedigital.core_banking.transactions.dto.TransactionDto;
+import at.deloittedigital.core_banking.transactions.mapper.TransactionMapper;
+import at.deloittedigital.core_banking.transactions.service.TransactionService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class TransactionController {
+
+    private final TransactionService transactionService;
+
+    private final TransactionMapper transactionMapper;
+
+    @GetMapping("/transactions")
+    public List<TransactionDto> getAllTransactions() {
```

Comment:
Would it be possible to add the filtering param for MVP0?
See "Services Documentation" on Azure wiki: `GET /transaction?accountId=1234&limit=100`

For the demo use-case we need to access transactions for a certain account, so imho it would be necessary to support this already in the first iteration.

Reply:
I left it out, as I wasn't sure where to draw the boundary between basic and non-basic data access services. Maybe all the filtering options deserve their own task, considering the size of the PR already.

Reply:
Separate PR is totally fine, however, we will need at least the account id filtering, otherwise we cannot gather data for a single account id for the MVP0 example shown in "Service Documentation". I assume with JpaRepository this could be rather simple addition using `findBy`

Reply:
I split the user story into separate tasks. "Filter transactions by account ID" and "align database schema changes" are two of them.